tests/unit/dfem/test_diffusion.cpp

```diff
@@ -255,8 +255,8 @@ void DFemDiffusion(const char *filename, int p, const int r)
          DOperator dop_mf(vsol, {{Coords, mfes}}, pmesh);
          const auto mf_vector_diffusion_qf =
             [] MFEM_HOST_DEVICE (const tensor<dscalar_t, DIM, DIM> &dudxi,
-               const tensor<real_t, DIM, DIM> &J,
-               const real_t &w)
+                                 const tensor<real_t, DIM, DIM> &J,
+                                 const real_t &w)
```

Comment:
This was not detected as not formatted properly because the `dfem` subdirectory of `tests/unit` is not in the list `UNIT_TESTS_SUBDIRS` defined in the main `makefile`.

It still needs to be added there.

Reply:
Agreed